controllers/grid/queries/QueryNotesGridCellProvider.inc.php

```diff
@@ -73,7 +73,7 @@ function getCellActions($request, $row, $column) {
 				}
 				return $actions;
 		}
-		return parent::getCellActions($request, $row, $column);
+		return parent::getCellActions($request, $row,, $column, $position);
```

Comment:
This looks like a typo.

Reply:
@crism, it looks like the tests have passed on the three PRs here, but I can't merge until this comment gets resolved ^

Reply:
Sorry, I looked for that before sending the PR, and still somehow missed it.

Reply:
Fixed.